### PR TITLE
Execution started event

### PR DESCRIPTION
**Pull request by CDimonaco**
Add the ExecutionStarted event from contracts, published to the `results` queue of rabbitmq.

This event will later by consumed by the web dashboard.

cc @jamie-suse

### CODE REVIEW

File: lib/wanda/messaging/mapper.ex [hunk 1]
```diff
@@ -18,9 +18,19 @@ defmodule Wanda.Messaging.Mapper do
     FactRequest,
     FactsGathered,
     FactsGatheringRequested,
-    FactsGatheringRequestedTarget
+    FactsGatheringRequestedTarget,
+    ExecutionStarted
   }
 
+  @spec to_execution_started(String.t(), String.t(), Target.t()) :: ExecutionStarted.t()
```
Comment (arbulu89) [inline lib/wanda/messaging/mapper.ex:25]: `Target` should be a list -> `[Target.t()]`

File: lib/wanda/messaging/mapper.ex [hunk 2]
```diff
@@ -84,6 +94,10 @@ defmodule Wanda.Messaging.Mapper do
     }
   end
 
+  defp map_target(%Target{agent_id: agent_id, checks: target_checks}) do
```
Comment (arbulu89) [inline lib/wanda/messaging/mapper.ex:97]: Strange the conversion from `checks` to `target_checks` hehe
But I guess is a copy/paste from somewhere else

File: mix.exs [hunk 3]
```diff
@@ -109,7 +109,7 @@ defmodule Wanda.MixProject do
       {:credentials_obfuscation, "3.0.0", override: true},
       {:jason, "~> 1.3"},
       {:yaml_elixir, "~> 2.9"},
-      {:trento_contracts, github: "trento-project/contracts", ref: "39abbf1", sparse: "elixir"},
+      {:trento_contracts, github: "trento-project/contracts", sparse: "elixir", ref: "1bff79a"},
```
Comment (arbulu89) [inline mix.exs:112]: I guess this is temporal until we merge the contract PR, right?

**Review by arbulu89 — COMMENTED** [review]
Nice! I would wait until we merge the contracts repo to put the commit of the merge.
Besides that, all fine

File: mix.exs (hunk 3 above)
Comment (CDimonaco) [reply mix.exs:112]: yes!

File: lib/wanda/messaging/mapper.ex (hunk 2 above)
Comment (CDimonaco) [reply lib/wanda/messaging/mapper.ex:97]: 👀  you're right

File: test/wanda/messaging/mapper_test.exs [hunk 4]
```diff
@@ -14,9 +14,35 @@ defmodule Wanda.Messaging.MapperTest do
...
+    targets = [
+      %Executions.Target{agent_id: "agent_1", checks: ["check_1", "check_2"]},
+      %Executions.Target{agent_id: "agent_2", checks: ["check_3"]}
+    ]
+
+    assert %ExecutionStarted{
+             execution_id: ^execution_id,
+             group_id: ^group_id,
+             targets: [
+               %Trento.Checks.V1.Target{
```
Comment (fabriziosestito) [inline test/wanda/messaging/mapper_test.exs:34]: let's alias these

**Review by fabriziosestito — APPROVED** [review]
LGTM, :shipit:  when the linter is happy + nitpick

File: mix.exs (hunk 3 above)
Comment (arbulu89) [reply mix.exs:112]: Man, at the end you didn't wait until merging first the other to put here the merge commit XD
Now this commit doesn't even exist in the `main` branch, which looks difficult to debug

**Pull request merged**